futures-core/src/lib.rs

```diff
@@ -20,15 +20,13 @@ compile_error!("The `cfg-target-has-atomic` feature requires the `unstable` feat
 extern crate alloc;
 
 pub mod future;
-#[doc(hidden)] pub use self::future::{Future, FusedFuture, TryFuture};
```

Comment:
Not sure if I agree that removing these is uncontroversial-- I use these imports frequently.

Reply:
Reverted traits related changes.